ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```
@@ -768,12 +786,12 @@ else
       source "${SHARED_DIR}/unset-proxy.sh"
     fi
     # This is a temporary conversion of cluster operator status to JSON matching the upgrade - may be moved to code in the future
-    curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 >/tmp/jq && chmod ug+x /tmp/jq
+    curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 >${ARTIFACT_DIR}/jq && chmod ug+x ${ARTIFACT_DIR}/jq
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)